idea/testData/quickfix/addInitializer/memberExtensionProperty.fir.kt

```diff
@@ -0,0 +1,10 @@
+// "Add initializer" "false"
```

Comment:
It would work. I am not sure why there are these class type based assertions in the test files. @darthorimar do you know why? Should we try replacing them with assertion on the message whenever we encounter these?

Reply:
Let's change the first line in FE1.0 testdata to avoid testdata duplication

Reply:
Cool, will do!